src/components/Icon.tsx

```
@@ -187,136 +187,171 @@ export function BrandingPrideIcon(props: SVGProps<SVGSVGElement>) {
-		);
-	}
+  if (resolvedTheme == "dark") {
+    return (
+      <svg
+        width="265"
+        height="266"
+        viewBox="0 0 265 266"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        {...props}
+      >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)